src/bridge/cockpitpolkitagent.c

```diff
@@ -384,13 +381,9 @@ cockpit_polkit_agent_initiate_authentication_finish (PolkitAgentListener *listen
                                                      GAsyncResult *res,
                                                      GError **error)
 {
-  g_warn_if_fail (g_simple_async_result_is_valid (res, G_OBJECT (listener),
-                  cockpit_polkit_agent_initiate_authentication));
+  g_warn_if_fail (g_task_is_valid (res, G_OBJECT (listener)));
```

Comment:
You can drop the `G_OBJECT()` cast.  GTask (wisely) takes `gpointer` here.

Reply:
Nice! done.